internal/pkg/service/buffer/worker/distribution/node.go

```diff
@@ -80,7 +81,9 @@ func NewNode(d dependencies, opts ...Option) (*Node, error) {
 	}
 
 	// Create etcd session
-	if err := n.createSession(); err != nil {
+	var err error
+	n.session, err = etcdclient.CreateConcurrencySession(n.logger, n.proc, n.client, c.ttlSeconds)
+	if err != nil {
```

Comment:
`etcd session` sluzi na vytvaranie klucov s `lease`, to znamena, ze ked sa klient odhlasi alebo sa prerusi spojenie, tak etcd server po `ttlSeconds` kluc automaticky zmaze.

Pouzivame to pre worker nodes discovery a potrebujem to aj inde, preto som vytvoril `CreateConcurrencySession` aby to bolo jednotne.